### PR TITLE
Feat[TEO-778] - View NGO Profile

### PR DESCRIPTION
**Pull request by biggicode**
[[TEO-778]](https://wearetribus.atlassian.net/browse/TEO-778)

### CODE REVIEW

File: mobile/src/assets/svg/clock.js
Comment (birloiflorian) [inline mobile/src/assets/svg/clock.js:1]: Please use the icon from feather

File: mobile/src/components/EventPresenter.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  title: string;
+  date: string;
+  location: string;
+  divison: string;
+  onPress: () => void;
+}
+
+const EventPresenter = ({ title, date, location, divison, onPress }: EventPresenterProps) => {
+  return (
+    <TouchableHighlight onPress={onPress} underlayColor="#F9F9F9">
```
Comment (birloiflorian) [inline mobile/src/components/EventPresenter.tsx:21]: Please use the underlayColor from theme.

File: mobile/src/components/EventPresenter.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+interface EventPresenterProps {
+  title: string;
+  date: string;
+  location: string;
+  divison: string;
+  onPress: () => void;
+}
+
+const EventPresenter = ({ title, date, location, divison, onPress }: EventPresenterProps) => {
```
Comment (birloiflorian) [inline mobile/src/components/EventPresenter.tsx:19]: This should be the EventItem

File: mobile/src/components/IconSvg.tsx [hunk 3]
```diff
@@ -4,8 +4,11 @@ import { SvgXml } from 'react-native-svg';
 interface IconSvgProps {
   size?: number;
   icon: string | null;
+  fill?: string;
 }
 
-const IconSvg = ({ icon, size }: IconSvgProps) => <SvgXml xml={icon} width={size} height={size} />;
+const IconSvg = ({ icon, size, fill = '#9CA3AF', ...svgProps }: IconSvgProps) => (
```
Comment (birloiflorian) [inline mobile/src/components/IconSvg.tsx:10]: Same here please use the fill color from the theme

File: mobile/src/components/Statistics.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const Statistics = () => {
           onPress={() => console.log('statistic comp 1 pressed')}
         />
         <StatisticsCard
-          icon={<IconSvg icon={ClockSvg} size={56} />}
```
Comment (birloiflorian) [inline mobile/src/components/Statistics.tsx:25]: Fill color from themeing

File: mobile/src/screens/OrganizationProfile.tsx [hunk 5]
```diff
@@ -9,12 +34,72 @@ const OrganizationProfile = ({ navigation }: any) => {
...
+  const onEventPress = () => {
+    console.log('event pressed');
+  };
+
   return (
-    <PageLayout title="Profil Organizatie" onBackButtonPress={navigation.goBack}>
-      <Text category="h1">OrganizationProfile</Text>
-      <Button onPress={onJoinOrganizationButtonPress}>Join</Button>
+    <PageLayout title={i18n.t('organization_profile:title')} onBackButtonPress={navigation.goBack}>
+      <ScrollView>
```
Comment (birloiflorian) [inline mobile/src/screens/OrganizationProfile.tsx:43]: Use the ScrollViewLayout component here

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: mobile/src/screens/OrganizationProfile.tsx (hunk 5 above)
Comment (biggicode) [reply mobile/src/screens/OrganizationProfile.tsx:43]: This needs discussion

Comment (birloiflorian) [reply mobile/src/screens/OrganizationProfile.tsx:43]: we leave it like this for now

**Review by birloiflorian — APPROVED** [review]

**Pull request merged**